mmv1/products/vertexai/FeatureOnlineStoreFeatureview.yaml

```
@@ -118,3 +124,62 @@ properties:
         description: |
           Columns to construct entityId / row keys. Start by supporting 1 only.
         item_type: Api::Type::String
+  - !ruby/object:Api::Type::NestedObject
+    name: 'vectorSearchConfig'
+    description: |
+      Configuration for vector search. It contains the required configurations to create an index from source data, so that approximate nearest neighbor (a.k.a ANN) algorithms search can be performed during online serving.
+    immutable: true
+    min_version: beta
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'embeddingColumn'
+        required: true
+        description: |
+          Column of embedding. This column contains the source data to create index for vector search. embeddingColumn must be set when using vector search.
+      - !ruby/object:Api::Type::Array
+        name: 'filterColumns'
+        description: |
+          Columns of features that're used to filter vector search results.
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::String
+        name: 'crowdingColumn'
+        description: |
+          Column of crowding. This column contains crowding attribute which is a constraint on a neighbor list produced by nearest neighbor search requiring that no more than some value k' of the k neighbors returned have the same value of crowdingAttribute.
+      - !ruby/object:Api::Type::Enum
+        name: distanceMeasureType
+        description: |
+          The distance measure used in nearest neighbor search.
+          SQUARED_L2_DISTANCE: Euclidean (L_2) Distance.
+
+          COSINE_DISTANCE: Cosine Distance. Defined as 1 - cosine similarity.
+
+          DOT_PRODUCT_DISTANCE: Dot Product Distance. Defined as a negative of the dot product.
+        values:
+          - :SQUARED_L2_DISTANCE
+          - :COSINE_DISTANCE
+          - :DOT_PRODUCT_DISTANCE
+      - !ruby/object:Api::Type::NestedObject
+        name: 'treeAhConfig'
+        exactly_one_of:
+          - treeAhConfig
+          - bruteForceConfig
+        description: |
+          Configuration options for the tree-AH algorithm (Shallow tree + Asymmetric Hashing).
```

Not sure if this is useful to include but it's in the API docs

```suggestion
          Configuration options for the tree-AH algorithm (Shallow tree + Asymmetric Hashing).
          Please refer to this paper for more details: https://arxiv.org/abs/1908.10396
```